### PR TITLE
Setting NewViewNumber to static get

### PR DESCRIPTION
**Pull request by vncoelho**
@igormcoelho @erikzhang @jsolman and @shargon 
I think that this will be enough.

Please fix the static, I think this is not the safer way.

### CODE REVIEW

File: neo/Consensus/ChangeView.cs [hunk 1]
```diff
@@ -4,7 +4,10 @@ namespace Neo.Consensus
 {
     public class ChangeView : ConsensusMessage
     {
-        public byte NewViewNumber;
+        /// <summary>
+        /// NewViewNumber is always set to the current ViewNumber asking changeview + 1
+        /// </summary>
+        public byte NewViewNumber => ViewNumber++;
```
Comment (erikzhang) [inline neo/Consensus/ChangeView.cs]:
```suggestion
        public byte NewViewNumber => ViewNumber + 1;
```

Comment (vncoelho) [reply neo/Consensus/ChangeView.cs]: I tried here, it says "Use block body for properties"

Comment (erikzhang) [reply neo/Consensus/ChangeView.cs]: `ViewNumber++` will change the value of `ViewNumber`.

Comment (vncoelho) [reply neo/Consensus/ChangeView.cs]: kkkkkkkkk, vishhh

Comment (vncoelho) [reply neo/Consensus/ChangeView.cs]: `++ViewNumber` ?

Comment (vncoelho) [reply neo/Consensus/ChangeView.cs]: "Consensus/ChangeView.cs(10,38): error CS0266: Cannot implicitly convert type 'int' to 'byte'. An explicit conversion exists (are you missing a cast?) [/opt/neoLib/neo/neo.csproj]"

Maybe we need that Get functions.

Comment (erikzhang) [reply neo/Consensus/ChangeView.cs]: public byte NewViewNumber => (byte)(ViewNumber + 1);

**Pull request merged**